src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -1279,28 +1294,28 @@ class FrontendGenImpl {
   }
 
   /*!
-   * Import output tensor, by doing the following:
-   * - Add the t/yp this output tensor to a list of tensor
+   * Import output value, by doing the following:
+   * - Add the type of this output tensor to a list of
    *   types representing return types of this graph function.
-   * - Add this output tensor to the list of Value
+   * - Add this output value to the list of values
    *   to be returned by the function representing computation graph.
-   * @param output onnx output tensor ValueInfoProto.
-   * @param ret_types a vector of tensor types representing graph's
-   *   output tensor types.
-   * @param ret_vals a vector of mlir Value  representing graph's
-   *   output tensor.
+   * @param output onnx output ValueInfoProto.
+   * @param ret_types a vector of types representing graph's output types.
+   * @param ret_vals a vector of mlir Value representing graph's output.
    */
   void ImportOutputTensor(const onnx::ValueInfoProto &output,
       llvm::SmallVectorImpl<Type> &ret_types,
       llvm::SmallVectorImpl<Value> &ret_vals) {
-    Value tensor_val = frontend_symbols_.GetTensorByOnnxName(output.name());
+    Value val = frontend_symbols_.GetTensorByOnnxName(output.name());
```

Comment:
is it ok if I rename `GetTensorByOnnxName`? it returns other than values than tensors (and returns types in some other instantiations of SymbolTable)

Reply:
Yes, you can rename the function.